.github/workflows/validate.yml

```diff
@@ -0,0 +1,24 @@
+name: Validate
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+      - "release/v*"
+    tags:
+      - "v*"
```

Comment:
Don't we want to run `validate` on all pushes?

```suggestion
```

Reply:
Good question, I left it as it was on Drone, but maybe we should just run on all pushes